lib/live_debugger/gen_servers/callback_tracing_server.ex

```diff
@@ -79,6 +68,16 @@ defmodule LiveDebugger.GenServers.CallbackTracingServer do
   # Because of that we do it asynchronously to speed up tracer a bit
   # We do not persist this trace because it is not displayed to user
   @spec handle_trace(term(), n :: integer()) :: integer()
+  defp handle_trace({_, _, :return_from, {Mix.Tasks.Compile.Elixir, _, _}, _, _}, n) do
+    Process.sleep(100)
+    add_live_modules_to_tracer()
+    n
+  end
+
+  defp handle_trace({_, _, _, {Mix.Tasks.Compile.Elixir, _, _}, _, _}, n) do
```

Comment:
```suggestion
  defp handle_trace({_, _, _, {Mix.Tasks.Compile.Elixir, _, _}, _}, n) do
```
Call trace has no return value